src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/ExceptionRecordMapper.java

```diff
@@ -19,7 +25,24 @@ public ExceptionRecord mapEnvelope(Envelope envelope) {
             envelope.jurisdiction,
             getLocalDateTime(envelope.deliveryDate),
             getLocalDateTime(envelope.openingDate),
-            mapDocuments(envelope.documents)
+            mapDocuments(envelope.documents),
+            mapOcrData(envelope)
         );
     }
+
+    private List<CcdCollectionElement<CcdKeyValue>> mapOcrData(Envelope envelope) {
+        if (envelope.ocrData != null) {
+            return envelope
+                .ocrData
+                .entrySet()
+                .stream()
+                .map(entry ->
+                    new CcdCollectionElement<>(
+                        new CcdKeyValue(entry.getKey(), entry.getValue())
+                    )
+                ).collect(toList());
+        } else {
+            return null;
```

Comment:
Still not favouring null for a collection field, but if we definitely sure it's the right way of implementation - it's fine (I did not just say that 🙊)

Reply:
From what I've been told, that's going to make a difference in CCD's UI (presence of an empty section or the lack of it)